### PR TITLE
Stub In Initial Files For Repo Template

### PR DESCRIPTION
**Pull request by seanmalloy**
* README
* Issue and PR templates
* Contributing guide lines
* License (subject to change)
* Travis config file

### CODE REVIEW

**Review by bjaroszewski — APPROVED** [review]
Looks good

File: README.md [hunk 1]
```diff
@@ -1 +1,17 @@
...
+Add steps for installing here.
+
+## Documentation
+
+Add documentaiton here, or add a link to
+the full documentation.
+
+## Developing
+
+See [CONTRIBUTING.md](https://github.com/KohlsTechnology/repo-template/blob/master/.github/CONTRIBUTING.md)
```
Comment (Smiley73) [inline README.md]: Make the path relative.
```suggestion
See [CONTRIBUTING.md](.github/CONTRIBUTING.md)
```

Comment (seanmalloy) [reply README.md]: Fixed

**Comment by seanmalloy** [comment]
@Smiley73 and @bjaroszewski I have made the suggested changes. Please review again. I'll have a follow pull request after this one gets merged.

**Pull request merged**